recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -0,0 +1,260 @@
+            "config",
+            "cur",
+            "datapipeline",
+            "dax",
+            "devicefarm",
+            "directconnect",
+            "discovery",
+            "dlm",
+            "dms",
+            "docdb",
+            "ds",
+            "dynamodb",
+            "dynamodbstreams",
+            "ec2",
+            "ecr",
+            "ecs",
+            "eks",
+            "elasticache",
+            "elasticbeanstalk",
+            "elasticfilesystem",
+            "elasticloadbalancing",
+            "elasticloadbalancingv2",
+            "elasticmapreduce",
+            "elastictranscoder",
+            "email",
+            "es",
+            "events",
+            "firehose",
+            "fms",
+            "fsx",
+            "gamelift",
+            "glacier",
+            "globalaccelerator",
+            "glue",
+            "greengrass",
+            "guardduty",
+            "health",
+            "iam",
+            "identity-management",
+            "importexport",
+            "inspector",
+            "iot-data",
+            "iot-jobs-data",
+            "iot",
+            "kafka",
+            "kinesis",
+            "kinesisanalytics",
+            "kinesisvideo",
+            "kms",
+            "lambda",
+            "lex",
+            "lightsail",
+            "logs",
+            "machinelearnings",
+            "macie",
+            "marketplace-entitlement",
+            "marketplacecommerceanalytics",
+            "mediaconvert",
+            "medialive",
+            "mediapackage",
+            "mediastore",
+            "mediatailor",
+            "meteringmarketplace",
+            "mobileanalytics",
+            "monitoring",
+            "mq",
+            "mturk-requester",
+            "neptune",
+            "opsworks",
+            "opsworkscm",
+            "organizations",
+            "pinpoint",
+            "polly",
+            "pricing",
+            "queues",
+            "quicksight",
+            "ram",
+            "rds",
+            "redshift",
+            "recognition",
+            "resource-groups",
+            "robomaker"
+            "route53",
+            "route53domains",
+            "s3",
+            "sagemaker",
+            "sdb",
+            "serverlessrepo"
+            "servicecatalog",
+            "servicediscovery",
+            "shield",
+            "signer",
+            "sms",
+            "snowball",
+            "sns",
+            "sqs",
+            "ssm",
+            "states",
+            "storagegateway",
+            "sts",
+            "support",
+            "swf",
+            "text-to-speech",
+            "texttract",
+            "transcribe",
+            "transfer",
+            "translate",
+            "waf",
+            "workdocs",
+            "worklink",
+            "workmail",
+            "workspaces",
+            "xray"
+           )
+    options = {
+            **{ x: [True, False] for x in sdks},
```

Comment:
some sdks components can't be disabled depending on which others sdks components are enabled. You'll have to track those dependencies to compute the proper package_id.
I guess that there is a lot of interdependencies between those components, which should be translated in `package_info()`

Reply:
see this comment https://github.com/conan-io/conan-center-index/pull/4401#discussion_r598355503